src/head/Head.test.js

```diff
@@ -119,4 +120,49 @@ describe("Head", () => {
 
         expect(onSort.calledWith(columns[0])).to.equal(true);
     });
+
+    it("should render title as string", () => {
+        const columns = [
+            {
+                title: "Column1",
+                sorted: "ASC"
+            }
+        ];
+        const newProps = Object.assign({}, props, {
+            columns
+        });
+        const wrapper = mount(<Head {...newProps} />, options);
+
+        expect(wrapper.find(Column).text()).to.contains("Column1")
+    });
+
+    it("should render title as functional Component", () => {
```

Comment:
Description is misleading - you passed react element, not a functional component, so description should be changed or change title to `title: () => <p>...</p>`